feedback_sentry/lib/feedback_sentry.dart

```diff
@@ -59,20 +55,23 @@ OnFeedbackCallback sendToSentry({
 
   return (UserFeedback feedback) async {
     await realHub.captureFeedback(
-      SentryFeedback(
-        contactEmail: email,
-        name: name,
-        message: feedback.text,
-        unknown: feedback.extra,
-      ),
-      hint: Hint.withScreenshot(
-        SentryAttachment.fromUint8List(
-          feedback.screenshot,
-          'screenshot.png',
-          contentType: 'image/png',
+        SentryFeedback(
+          contactEmail: email,
+          name: name,
+          message: feedback.text,
         ),
-      ),
-    );
+        hint: Hint.withScreenshot(
+          SentryAttachment.fromUint8List(
+            feedback.screenshot,
+            'screenshot.png',
+            contentType: 'image/png',
+          ),
+        ), withScope: (scope) {
+      scope.setContexts(
+        "user_feedback",
+        feedback.extra,
+      );
```

Comment:
Why can't we use `SentryFeedback` just like before?

Reply:
it didn't allow me use the `unknown` var in the test (https://github.com/ueman/feedback/actions/runs/15869542084/job/44789906814)
maybe its unrelated